src/cases/containers/case-search/case-search.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit, ViewEncapsulation, OnDestroy } from '@angular/core';
 import { Store, select } from '@ngrx/store';
 import * as fromCasesFeature from '../../store';
-import { Jurisdiction, CaseType, CaseState, SearchResultView, PaginationMetadata } from '@hmcts/ccd-case-ui-toolkit';
+import { Jurisdiction, CaseType, CaseState, SearchResultView, PaginationMetadata, SearchResultViewItem } from '@hmcts/ccd-case-ui-toolkit';
```

Comment:
If the "SearchResultViewItem" import is new code, is it being used in this file? If not please remove it.

Reply:
done